packages/db/queries/get_active_clusters.sql

```diff
@@ -0,0 +1,4 @@
+-- name: GetActiveClusters :many
+SELECT DISTINCT sqlc.embed(c)
```

Comment:
why is there distinct if cluster <-> team should be unique?

Reply:
Added migrations to remove the unique index in the team table. So now we can have multiple teams with the same cluster ID, there are no blockers for this.